### PR TITLE
Remove inactive strategies

### PR DESCRIPTION
**Pull request by drondin**
Remove OneInch strats

### CODE REVIEW

File: src/app/static/static-values.ts [hunk 1]
```diff
@@ -62,10 +62,6 @@ export class StaticValues {
     'SUSHI_MIC_USDT',
     'SUSHI_MIS_USDT',
     'CRV_OBTC',
-    'ONEINCH_ETH_DAI',
-    'ONEINCH_ETH_USDC',
```
Comment (belbix) [inline src/app/static/static-values.ts:66]: in this list, we have all strategies
we can have withdrawals from an old one

File: src/app/static/static-values.ts [hunk 2]
```diff
@@ -164,10 +160,6 @@ export class StaticValues {
     'SUSHI_ETH_UST',
     // 'SUSHI_MIC_USDT',
     // 'SUSHI_MIS_USDT',
-    'ONEINCH_ETH_DAI',
-    'ONEINCH_ETH_USDC',
```
Comment (belbix) [inline src/app/static/static-values.ts:168]: do we have any strategies instead of it?

File: src/app/static/static-values.ts (hunk 1 above)
Comment (drondin) [reply src/app/static/static-values.ts]: reverted

File: src/app/static/static-values.ts (hunk 2 above)
Comment (drondin) [reply src/app/static/static-values.ts:168]:
```
    'ETH_DAI_HODL',
    'ETH_USDC_HODL',
    'ETH_USDT_HODL',
    'ETH_WBTC_HODL',
```

are the new ones

File: src/app/static/static-values.ts [hunk 3]
```diff
@@ -225,10 +225,6 @@ export class StaticValues {
       case 'PS':
       case 'iPS':
         return 'FARM';
-      case 'CRV_LINK':
```
Comment (belbix) [inline src/app/static/static-values.ts:228]: why? :)

Comment (paurosello) [reply src/app/static/static-values.ts:228]: This is a mistake, not sure what happened

Comment (drondin) [reply src/app/static/static-values.ts:228]: ouch, fixing right now

**Comment by drondin** [comment]
History data takes vaults from the "vaults" attribute so vaults are only hidden from the front page, not from download historic data

**Pull request merged**